test-data/test-etree.yml

```diff
@@ -37,3 +37,21 @@
         parser = etree.XMLParser()
         element = parser.makeelement("foobar")
         reveal_type(element)  # N: Revealed type is 'lxml.etree._Element'
+-   case: etree_tostring_bytes
+    disable_cache: true
+    main: |
+        from lxml import etree
+        string = etree.tostring(etree.Element("foo"))
+        reveal_type(string)  # N: Revealed type is 'builtins.bytes'
```

Comment:
Since you're explicitly listing some encodings above, I'd expect at least a test for one of them, maybe also for one that does not match the list.

Reply:
Sure, I've added those. Should I squash the changes?